### PR TITLE
Fixes #8316: Add a flag file to tell the agent not to restart cf-serverd

### PR DESCRIPTION
**Pull request by jooooooon**
https://www.rudder-project.org/redmine/issues/8316

### CODE REVIEW

**Comment by jooooooon** [comment]
Commit modified

**Comment by peckpeck** [comment]
I'm OK with this patch, but shouldn't it be on older branches too ?
Since the technique can on a recent agent from an older server.

**Comment by peckpeck** [comment]
This PR is nor mergeable anymore

File: techniques/system/common/1.0/update.st [hunk 1]
```diff
@@ -367,7 +367,7 @@ bundle agent update_action
         action  => u_ifwin_bg,
         classes => outcome("executor");
 
-    start_server::
+    !should_disable_server.start_server::
```
Comment (ncharles) [inline techniques/system/common/1.0/update.st]: i think it should go as far as stopping effectively the cf-serverd component

**Comment by jooooooon** [comment]
Commit modified

**Pull request merged**